src/coreComponents/physicsSolvers/fluidFlow/IsothermalCompositionalMultiphaseFVMKernels.hpp

```diff
@@ -436,6 +436,8 @@ class FaceBasedAssemblyKernel : public FaceBasedAssemblyKernelBase
   FaceBasedAssemblyKernel( integer const numPhases,
                            globalIndex const rankOffset,
                            integer const hasCapPressure,
+                           integer const useC1PPU,
```

Comment:
I think you have then to update accordingly in[ ``createAndLaunch()`` ](https://github.com/GEOS-DEV/GEOS/pull/2410/files#diff-9b6dc47db353c661c4caf325bdbf39b3128219459054bf1ec6363a9e563e22f1R1020-R1055)

Reply:
Yes, looks like I forgot to commit that... let me fix it.